package.json

```diff
@@ -4,14 +4,23 @@
   "description": "A Social platform for the creative at heart",
   "main": "index.js",
   "scripts": {
-    "test": "npm run test:migrate && NODE_ENV=test nyc --reporter=html --reporter=text mocha --require @babel/register --require babel-polyfill ./test/ --recursive --exit",
+    "test": "npm run test:migrate && NODE_ENV=test nyc --reporter=html --reporter=text mocha --require @babel/register --require babel-polyfill ./server/test/ --recursive --exit",
     "migrate": "sequelize db:migrate",
     "undo-all:migrate": "sequelize db:migrate:undo:all",
+    "refreshdb": "npm run undo-all:migrate && npm run migrate",
     "undo-current:migrate": "sequelize db:migrate:undo",
     "test:migrate": "NODE_ENV=test npm run undo-all:migrate && NODE_ENV=test npm run migrate ",
-    "start": "babel-node ./index.js",
+    "start": "node --require babel-polyfill ./build/index.js",
+    "start:dev": "nodemon --exec babel-node ./server/index.js",
```

Comment:
Hi @Tyak99 kindly share some light as to why you're using babel-node?

Reply:
@waleCloud we are using babel-node on development to transpile the new es6 syntax